### ISSUE TITLE
Make the Python version which GraalVM is targetting more prominent

### ISSUE THREAD

**Issue by ambv** [oracle/graal#352]
Hi! Exciting project.

Instead of addressing the language as just "Python" in the docs, it would be helpful to state "Python 3" which will answer the most pressing question Python users have. I especially mean the README here and the landing page on https://graalvm.org.

While the "Reference manual for Python" already states GraalVM is targetting Python 3.7, that's buried a bit.

(BTW, great decision to start with 3.7!)

**Comment by lukasstadler** [comment]
Indeed, we should be more clear about that, it's not obvious that we're always referring to Python 3.
Thanks for the hint!
